aireview/src/aireview/domain/services/code_analyzer.py

```diff
@@ -16,7 +16,9 @@ def analyze_code(self, pull_request: PullRequest) -> Review:
         # self._ai_agent.initialize(f"Creating context for review of pull request {pull_request.number}")
         for index, file in enumerate(pull_request.files):
             prompt = self._build_analysis_prompt(file)
+            print(f"prompt: {prompt}")
             analysis = self._ai_agent.analyse(prompt)
```

Comment:
**style**: Debug print statements should be removed from production code. Consider using a proper logging framework like Python's built-in 'logging' module instead.